ClientApp/src/App.vue

```diff
@@ -49,7 +49,7 @@ export default class App extends Vue {
   private drawer: boolean = true;
   private miniVariant: boolean = false;
   private right: boolean = true;
-  private title: string = 'ASP.NET Core Vue Starter';
+  private title: string = 'MercuryHealth';
```

Comment:
Uh, shouldn't it be Mercury<space>Health?
```suggestion
  private title: string = 'Mercury Health';
```